vpr/src/route/connection_router.cpp

```diff
@@ -1049,15 +1055,9 @@ t_bb ConnectionRouter<Heap>::add_high_fanout_route_tree_to_heap(
 static inline bool relevant_node_to_target(const RRGraphView* rr_graph,
```

Comment:
Add a Doxygen comment for this function and its purpose at the point where it is declared.